src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
The trace is empty. The `$driverNames` property is called from the `getPlatform` method. This is the error I got:
`FatalErrorException in DatabaseExceptionTrait.php line 47: 
Error: Cannot access property Bolt\Exception\Database\DatabaseConnectionException::$driverNames`
To reproduce you can change the `databasename` in the `config.yml` to a non existing databasename and you should get the error.